tests/Container/ContainerTest.php

```diff
@@ -606,7 +605,7 @@ public function testContainerCanResolveClasses()
 
     // public function testContainerCanCatchCircularDependency()
```

Comment:
How is there a commented function in the repo?! :)

Reply:
> How is there a commented function in the repo?! :)

Ah, it was a @taylorotwell commit.. https://github.com/laravel/framework/commit/332844e5bde34f8db91aeca4d21cd4e0925d691e

#nevercommitcommentedoutcode